### ISSUE TITLE
ST 4.0.2 Cannot open license website (or any external website links)

### ISSUE THREAD

**Issue by RealRaven2000** [RealRaven2000/SmartTemplates#252]
Apparently when trying to renew the license (or upgrade) the external webbrowser does not open the shop link anymore.

![image](https://github.com/RealRaven2000/SmartTemplates/assets/6314674/0e12440b-5f2c-4e97-b780-7f4776ef0421)

Instead we see the following error in JavaScript error console:

```log
Uncaught ReferenceError: can't access lexical declaration 'uri' before initialization
    openLinkInBrowser chrome://smarttemplate4/content/smartTemplate-util.js:870
    goPro chrome://smarttemplate4/content/smartTemplate-register-dlg.js:304
    oncommand chrome://smarttemplate4/content/register.xhtml:1
2 smartTemplate-util.js:870:47

```

**Comment by RealRaven2000** [comment]
Quick Fix version:

[smartTemplate-fx-4.1pre14.zip](https://github.com/RealRaven2000/SmartTemplates/files/12300441/smartTemplate-fx-4.1pre14.zip)

----

To test the version above, download the zip file, drag the file into Thunderbird Add-ons Manager, _**do not extract contents**_ or it won't install.

**Comment by RealRaven2000** [comment]
Fixed in 4.1 - Published 09/08/2023
